content/docs/00-release-notes.md

```diff
@@ -127,18 +127,18 @@ Palette 3.4.0 has various security upgrades, better support for multiple Kuberne
 - Palette PDE is now available in self-hosted installation of Palette.
 
 
-- PDE now has a Command Line Interface (CLI) that you can use for programmatic access to PDE resources. Users can perform actions such as create, list, delete, resize, pause, and resume virtual cluster. You can also download the kubeconfig file of a virtual cluster with the CLI.
+- PDE now has a Command Line Interface (CLI) that you can use for programmatic access to PDE resources. Users can perform actions such as create, list, delete, resize, pause, and resume virtual cluster. You can also download the kubeconfig file of a virtual cluster with the CLI. Refer to the [Palette CLI](/devx/manage-dev-engine/install-palette-cli) documentation page to learn more.
 
 
 ## Improvements
 
-- Container applications that expose a service now automatically receive ingress support with HTTPS support out-of-the-box.
+- Container applications that expose a service now automatically receive ingress support with HTTPS support out-of-the-box. This means that exposed service URLs automatically default to HTTPS.
```

Comment:
This means that exposed service URLs automatically default to valid certificate-based dynamically generated DNS URLs.